x/validator/client/cli/tx.go

```diff
@@ -91,11 +88,20 @@ func GetCmdEditCandidateDescription(cdc *codec.Codec) *cobra.Command {
 
 			txr, err := transactor.NewCliTransactor(cdc, viper.GetString(flags.FlagHome))
 			if err != nil {
-				log.Error(err)
 				return err
 			}
 
-			msg := types.NewMsgEditCandidateDescription(args[0], description, txr.Key.GetAddress())
+			ksBytes, err := ioutil.ReadFile(viper.GetString(common.FlagEthKeystore))
+			if err != nil {
+				return err
+			}
+
+			key, err := keystore.DecryptKey(ksBytes, viper.GetString(common.FlagEthPassphrase))
```

Comment:
What if passphrase not provided in the config file? Here we only need the address, no need to DecryptKey